include/swift/Runtime/Portability.h

```diff
@@ -35,11 +35,12 @@ inline static int
 swift_asprintf(char **strp, const char *fmt, ...) {
   va_list args;
   va_start(args, fmt);
-#if defined(_WIN32)
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wuninitialized"
-  int len = _vscprintf(fmt, args);
-#pragma GCC diagnostic pop
+
```

Comment:
Can we call through to `swift_vasprintf` and remove the duplicate implementation?

Reply:
We can and we should! 👍